src/lang/Language.php

```diff
@@ -138,6 +165,17 @@ protected static function loadLang(string $path, string $languageCode) : array{
 		throw new LanguageNotFoundException("Language \"$languageCode\" not found");
 	}
 
+	public function merge(Language $language) : void{
```

Comment:
Is "merge" really a good way to look at this? Do we really want to override/affect existing language entries? If we expect both language objects to be disjoint, "add" sounds like a more reasonable verb.

Reply:
If it's to be renamed I think I prefer `importLanguageStrings` for accuracy

Reply:
`importLanguageStrings` doesn't tell anything about overwrite vs fill-the-blanks semantics.